usaspending_api/awards/migrations/0059_auto_20170207_1854.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-02-07 18:54
+from __future__ import unicode_literals
+
+import django.db.models.deletion
+from django.db import connection, migrations, models
+
+
+def attach_tas(apps, schema_editor):
+    with connection.cursor() as cursor:
```

Comment:
I've googled up claims that the "better" way to do this is with `migrations.RunSQL("UPDATE financial_accounts_by_awards...")` - though I'm not sure what they mean by "better" aside from saving one line of code and one import.  

I guess one other benefit is that you can define the *reverse* SQL in the same place instead of nesting it in a RunPython, FWIW.  Or (as in this case) just not define the reverse SQL, not wrap the RunSQL in a RunPython, and save maybe four lines.

https://markusholtermann.eu/2014/09/django-17-database-migrations-done-right/